Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeInteractionRegionLayers.mm

```diff
@@ -31,22 +31,77 @@
 #import "PlatformCALayerRemote.h"
 #import "RemoteLayerTreeHost.h"
 #import <QuartzCore/QuartzCore.h>
+#import <RealitySystemSupport/RealitySystemSupport.h>
 #import <pal/spi/cocoa/QuartzCoreSPI.h>
+#import <wtf/SoftLinking.h>
 
-#if USE(APPLE_INTERNAL_SDK)
-#import <WebKitAdditions/RemoteLayerTreePropertyApplierInteractionRegionAdditions.mm>
-#else
-static Class interactionRegionLayerClass() { return [CALayer class]; }
-static void configureLayerForInteractionRegion(CALayer *, NSString *) { }
-static void configureLayerAsGuard(CALayer *, NSString *) { }
-#endif
+//// FIXME: rdar://105775731
+SOFT_LINK_PRIVATE_FRAMEWORK_OPTIONAL(RealitySystemSupport)
+SOFT_LINK_CLASS_OPTIONAL(RealitySystemSupport, RCPGlowEffectLayer)
+SOFT_LINK_CONSTANT_MAY_FAIL(RealitySystemSupport, RCPAllowedInputTypesUserInfoKey, const NSString *)
+
+@interface CALayer ()
+@property (nonatomic) CGFloat sizeMultiplier;
+@property (nonatomic, copy) void (^effectGroupConfigurator)(CARemoteEffectGroup *group);
+@end
 
 namespace WebKit {
 using namespace WebCore;
 
 NSString *interactionRegionTypeKey = @"WKInteractionRegionType";
 NSString *interactionRegionGroupNameKey = @"WKInteractionRegionGroupName";
 
+#if PLATFORM(VISION)
```

Comment:
Do we need this conditional? Since `ENABLE(INTERACTION_REGIONS_IN_EVENT_REGION)` is equivalent to `PLATFORM(VISION)`.

Noticed that you didn't include any guards around `#import <RealitySystemSupport/RealitySystemSupport.h>`, so we should probably remove to be consistent (or also add a guard there).